components/monitoring/logging/rbac/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
You need to include this new rbac folder/kustomization in `monitoring/logging/base/kustomization.yaml` if you want this in all overlays or in `monitoring/logging/staging/kustomization.yaml`  and `monitoring/logging/production/kustomization.yaml`  for staging and prod only

Reply:
@hugares I moved it under base as I want to apply it for both staging and production.